packages/editor-ui/src/components/ButtonParameter/ButtonParameter.vue

```diff
@@ -227,7 +276,7 @@ onMounted(() => {
 
 <style module lang="scss">
 .input * {
-	border: 0 !important;
+	border: 1.5px transparent !important;
```

Comment:
can this be done without !important ?

Reply:
no, we need to override defaults of N8nInput(ElInput)